consoles/sshVirtsh.pm

```diff
@@ -352,9 +353,16 @@ sub _copy_image_vmware ($self, $name, $backingfile, $file_basename, $vmware_open
 }
 
 sub _copy_image_else ($self, $file, $file_basename, $basedir) {
-    $self->run_cmd(sprintf("rsync -av '$file' '$basedir/%s'", $file_basename)) && die 'rsync failed';
+    if (-e $file_basename && defined which 'rsync') {    # utilize asset possibly cached by openQA worker
```

Comment:
In which directory would this be called?

Reply:
I would assume that the working directory doesn't change. So it would still be isotovideo's working directory which is where the worker would create a link to the cached asset. So this hopefully works. I'm still testing this via https://github.com/os-autoinst/os-autoinst/blob/master/doc/backends.md#local-setup.